doc/users/plotting/examples/demo_gridspec04.py

```diff
@@ -34,7 +34,7 @@ def make_ticklabels_invisible(fig):
 ax6 = plt.Subplot(f, gs01[-1, -1])
 f.add_subplot(ax6)
 
-plt.suptitle("GirdSpec Inside GridSpec")
+plt.suptitle("GridSpec Inside GridSpec")
 make_ticklabels_invisible(plt.gcf())
```

Comment:
This should just be `f` instead of `plt.gcf()`. Not your fault, but if you want to fix it while you're here...